<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaSayfa</class>
 <widget class="QMainWindow" name="AnaSayfa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>781</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AnaSayfa</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Araç Ekle</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Araç Türü</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAracEkle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;
color: rgb(255, 255, 255);
background-color:rgb(0, 85, 255);</string>
     </property>
     <property name="text">
      <string>ARAÇ EKLE</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAracAdi">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxAraclar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>Otobüs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Metro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gemi</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Araç Adı</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ulaşım Ücreti</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxUlasimUcreti">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetAraclar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Araç Türü</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Araç Adı</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ulaşım Ücreti</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Durak Ekle</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Durak Türü</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDurakEkle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;
color: rgb(255, 255, 255);
background-color:rgb(0, 85, 255);</string>
     </property>
     <property name="text">
      <string>DURAK EKLE</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDurakAdi">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDuraklar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>Otobüs Durağı</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Metro İstasyonu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vapur İskelesi</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Durak Adı</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>411</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Hat ve Mesafe</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Duraklar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonHatEkle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;
color: rgb(255, 255, 255);
background-color:rgb(0, 85, 255);</string>
     </property>
     <property name="text">
      <string>HAT EKLE</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHatismi">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDurak1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hat İsmi</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDurak2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMesafe">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>50000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mesafe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Metre</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Gün Sonu</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hatlar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonGunSonu">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 8px;
color: rgb(255, 255, 255);
background-color:rgb(0, 85, 255);</string>
     </property>
     <property name="text">
      <string>GÜN SONU
KAYDET</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tarih</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxHatlar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxYolcuSayisi">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Yolcu Sayısı</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEditTarih">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2021</year>
       <month>6</month>
       <day>10</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2021</year>
       <month>6</month>
       <day>10</day>
      </date>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetDuraklar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Durak Türü</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Durak Adı</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetGunler">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>400</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Hat İsmi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tarih</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Yolcu Sayısı</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetHatlar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Hat İsmi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Durak 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Durak 2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mesafe</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
